### PR TITLE
Add kubemark jobs to source control.

### PR DESCRIPTION
**Pull request by spxtr**
cc @ixdy 
ref #18122

### CODE REVIEW

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -904,6 +904,7 @@ case ${JOB_NAME} in
     : ${E2E_CLUSTER_NAME:="kubernetes-kubemark"}
     : ${E2E_NETWORK:="kubernetes-kubemark"}
     : ${PROJECT:="k8s-jenkins-kubemark"}
+    : ${E2E_PUBLISH_GREEN_VERSION:="true"}
```
Comment (gmarek) [inline hack/jenkins/e2e.sh]: I recall @roberthbailey having some problems with that.

Comment (roberthbailey) [inline hack/jenkins/e2e.sh]: Yeah, kubemark shouldn't affect our latest.txt file.

File: hack/jenkins/e2e.sh [hunk 2]
```diff
@@ -923,6 +925,7 @@ case ${JOB_NAME} in
     : ${E2E_CLUSTER_NAME:="kubernetes-kubemark-500"}
     : ${E2E_NETWORK:="kubernetes-kubemark-500"}
     : ${PROJECT:="kubernetes-scale"}
+    : ${E2E_PUBLISH_GREEN_VERSION:="true"}
```
Comment (roberthbailey) [inline hack/jenkins/e2e.sh]: same here

File: hack/jenkins/e2e.sh [hunk 3]
```diff
@@ -942,6 +945,7 @@ case ${JOB_NAME} in
     : ${E2E_CLUSTER_NAME:="kubernetes-kubemark-scale"}
     : ${E2E_NETWORK:="kubernetes-kubemark-scale"}
     : ${PROJECT:="kubernetes-scale"}
+    : ${E2E_PUBLISH_GREEN_VERSION:="true"}
```
Comment (roberthbailey) [inline hack/jenkins/e2e.sh]: and here

**Comment by spxtr** [comment]
Okay, I removed those lines from `e2e.sh`. That parameter had previously been set in Jenkins config.

@gmarek PTAL

File: hack/jenkins/e2e.sh [hunk 4]
```diff
@@ -913,6 +913,7 @@ case ${JOB_NAME} in
     NUM_NODES="10"
     MASTER_SIZE="n1-standard-2"
     NODE_SIZE="n1-standard-1"
+    E2E_ZONE="asia-east1-a"
```
Comment (roberthbailey) [inline hack/jenkins/e2e.sh]: Does this move all of our jobs to asia?

Comment (spxtr) [inline hack/jenkins/e2e.sh]: `kubernetes-kubemark-gce` currently sets this in Jenkins config. For this PR, I've moved it out and into the case statement, and it only hits this job. Other jobs will remain where they are.

I'd be happy to change this if this job should be running elsewhere.  `kubernetes-e2e-gce-enormous-cluster` also runs in `asia-east1-a`.

**Comment by gmarek** [comment]
LGTM

**Comment by spxtr** [comment]
Manual merge to watch Jenkins.

**Pull request merged**